build/NuGetPackages/CreateNuGetPackages.proj

```diff
@@ -72,7 +72,7 @@
     This target is used to customize the versions generated by NerdBank.GitVersioning.  The $(PrereleaseVersion) is read from version.json and
     then the git commit height and build revision is appended.
   -->
-    <PropertyGroup>
+    <PropertyGroup Condition=" '$(PublicRelease)' == 'false' ">
```

Comment:
As a rule I prefer `!= true` and `== true` rather than dual explicit comparisons (allowing the unset case to default into one bucket or the other). But if NB.GV guarantees that it's explicitly set then this is fine.